contents/handbook/growth/sales/refunds.md

```diff
@@ -23,6 +23,8 @@ We issue refunds in this category if this is the first bill >$1 and/or they meet
 #### Next steps
 - Use [Refund - Learning Curve email template](https://posthog.lightning.force.com/lightning/r/EmailTemplate/00XHp000001vSDAMA2/view) to respond to the customer if they match criteria.
 - Assign the refund request to the RevOps team in Salesforce for processing.
+- If the issue is caught before the end of the current billing cycle, contact the customer and explain that after the current billing cycle we will refund the delta occurred by unusual usage during the learvning curve.
```

Comment:
Applying credits is sometimes easier depending on the nature of the problem and where they are in their billing cycle. Also some people never pay so applying credits for known issues before invoice is finalized prevents failed payments. can clarify like this "We either issue credits or provide a refund for the delta occurred due to unusual usage. Check with RevOps if you want to share the exact solution with your customer."